schema/yaml/graph.yml

```diff
@@ -10,100 +10,33 @@ output:
     description: 'Graph error'
     $ref: '/shared/message'
 
-  addnode:
-    id: 'output/addnode'
-    description: 'Add node to a graph.'
-    $ref: '../input/addnode'
```

Comment:
This is an incompatible change and I don't see the reason for it?`*`
The graph operations need to reply with something, and might as well reply with something specific. The reply sent should then contain the `id` of the request it is a reply to, and this can be used to correlate it.

`*` and we have like 7 implementations of the protocol, so we should to be causious with breaking things.